workflow/controller/controller.go

```diff
@@ -510,24 +508,6 @@ func (wfc *WorkflowController) processNextItem() bool {
 		wfc.throttler.Remove(key)
 		return true
 	}
-
```

Comment:
Moving inside `operate()` to support the `synchronization` from WfDefault and WFTSpec